addon/components/mobile/flexberry-objectlistview.js

```diff
@@ -138,4 +140,132 @@ export default FlexberryObjectlistview.extend({
     @default true
   */
   columnsWidthAutoresize: true,
+
+  /**
+    Array of objects corresponding to list of pages.
+
+    Each page is presented as object with following properties:
+    - **number** - Number of page.
+    - **isCurrent** - If `true` this page is current.
+    - **isShow** - If `true` this page showing in pages list.
+
+    @property mobilePages
+    @type Array
+    @readOnly
+  */
+  mobilePages: Ember.computed('pages', function() {
```

Comment:
Думаю это свойство лучше приватным сделать.
```suggestion
    @private
    @property _mobilePages
    @type Array
    @readOnly
  */
  _mobilePages: Ember.computed('pages', function() {
```